demos/shared/trace_streamer/stream_traces.py

```diff
@@ -0,0 +1,34 @@
+import os
+import time
+import requests
+
+otel_tracing_endpoint = os.getenv(
+    "OTEL_TRACING_HTTP_ENDPOINT", "http://localhost:4318/v1/traces"
+)
+envoy_log_path = os.getenv("ENVOY_LOG_PATH", "/var/log/envoy.log")
+
+print(f"Using otel-tracing host: {otel_tracing_endpoint}")
+print(f"Using envoy log path: {envoy_log_path}")
+
+
+def process_log_line(line):
+    try:
+        response = requests.post(
+            url=otel_tracing_endpoint,
+            data=line,
+            headers={"Content-Type": "application/json"},
+        )
+        print(f"Sent trace to otel-tracing: {response.status_code}")
+    except Exception as e:
+        print(f"Failed to send trace to otel-tracing: {e}")
+
+
+with open(envoy_log_path, "r") as f:
+    while True:
```

Comment:
I think best to check if the log file exists to begin with, if not that means tracing was off - then no need to run this for loop.

Reply:
it must exist - if it doesnt we should fail/crash

I also make sure to create this file in Dockerfile see here -https://github.com/katanemo/arch/pull/262/files#diff-8e3a6a6de1bde2a67bc1791e073f4724332070eac7b751f83122f8929eb87f1fR9